src/message_api.c

```diff
@@ -6268,23 +6274,15 @@ static PEP_STATUS _decrypt_message(
                             if (status == PEP_STATUS_OK && my_rev_ids) {
                                 // get identities in this list the message was to/cc'd to (not for bcc)
                                 identity_list* used_ids_for_key = NULL;
-                                status = ident_list_intersect(my_rev_ids, msg->to, &used_ids_for_key);
-                                if (status != PEP_STATUS_OK)
-                                    goto pEp_error; // out of memory
-
-                                identity_list* used_cc_ids = NULL;    
-                                status = ident_list_intersect(my_rev_ids, msg->cc, &used_cc_ids);
-                                if (status != PEP_STATUS_OK)
-                                    goto pEp_error;
-
-                                used_ids_for_key = identity_list_join(used_ids_for_key, used_cc_ids);
-                                
+                                used_ids_for_key = identity_list_add(used_ids_for_key, msg->from);
```

Comment:
Why include `msg->from` here? The reset message later is hard-coded _to_ `msg->from`.